frontend/components/Forms/SetPrice/SetPrice.js

```diff
@@ -0,0 +1,68 @@
+import { ErrorMessage, Field, Form, Formik } from "formik";
+import React, { useContext } from "react";
+import { FormContext } from "../../Overlay/CreateProject.js";
+import * as yup from "yup";
+import classes from "../../../styles/Forms.module.css";
+
+export default function SetPrice() {
+  const { activeStepIndex, setActiveStepIndex, formData, setFormData } =
+    useContext(FormContext);
+
+  const renderError = (message) => (
+    <p className={classes.errorMessage}>{message}</p>
+  );
+
+  const ValidationSchema = yup.object().shape({
+    SetPrice: yup
```

Comment:
nitpick: can you use lower case s for SetPrice here...
```suggestion
    setPrice: yup
```